python/ray/exceptions.py

```diff
@@ -280,6 +280,8 @@ def __init__(self, cause: Union[RayTaskError, ActorDiedErrorContext] = None):
         self._actor_init_failed = False
         # -- The base actor error message. --
         self.base_error_msg = "The actor died unexpectedly before finishing this task."
+        # Whether the node was preempted
+        self.preempted = False
```

Comment:
```suggestion
        self._preempted = False
```
And create a property to access it?